examples/Benchmark.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct 18 15:53:27 2016
+
+@author: Michael Wu
+
+Benchmark test
+Giving performances of RF(scikit) and MultitaskDNN(Keras & TF)
+on datasets: muv, nci, pcba, tox21
+
+"""
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+
+import sys
+import os
+import numpy as np
+import shutil
+import time
+
+from sklearn.linear_model import LogisticRegression
+from sklearn.ensemble import RandomForestClassifier
+
+from deepchem.datasets import Dataset
+from deepchem import metrics
+from deepchem.metrics import Metric
+from deepchem.utils.evaluate import Evaluator
+from deepchem.models.keras_models.fcnet import MultiTaskDNN
+from deepchem.models.keras_models import KerasModel
+from deepchem.models.multitask import SingletaskToMultitask
+from deepchem.models.sklearn_models import SklearnModel
+from deepchem.models.tensorflow_models.fcnet import TensorflowMultiTaskClassifier
+from deepchem.models.tensorflow_models import TensorflowModel
+from deepchem.splits import RandomSplitter
+
+from muv.muv_datasets import load_muv
+from nci.nci_datasets import load_nci
+from pcba.pcba_datasets import load_pcba
+from tox21.tox21_datasets import load_tox21
+
+def benchmarkLoadingDatasets(base_dir_o, n_features = 1024, datasetName = 'all',
+                             model = 'all', reload = True, verbosity = 'high'):
+  
+  assert datasetName in ['all', 'muv', 'nci', 'pcba', 'tox21']
+  
+  if datasetName == 'all':
+    #currently not including the nci dataset
+    datasetName = ['muv','pcba','tox21']
+  else:
+    datasetName = [datasetName]
+      
+  if 'tox21' in datasetName:
+    print('-------------------------------------')
+    print('Benchmark test on dataset: tox21')
+    print('-------------------------------------')
+    base_dir = os.path.join(base_dir_o, "tox21")
+    time1 = time.time()
```

Comment:
Can you rename `time1` as `time_start`? As a general stylistic comment, using variables names like `var1` isn't (usually) good python style? Would you mind renaming all variables in this style to use more descriptive names?